### PR TITLE
Upgrade to Rails 6

### PR DESCRIPTION
**Pull request by jbirdjavi**

### CODE REVIEW

**Comment by jbirdjavi** [comment]
@Omicron7 I had to upgrade activeadmin and do some javascript rearranging with this.  I believe everything is working correctly, but I think it would be helpful if you could test this (or if you know of someone else that should test, that'd be fine too).  Since there's no staging site for this app (that I know of), I can provide a link to my local server if needed.

**Comment by jbirdjavi** [comment]
@Omicron7 I found an issue with how the campaign-form.js file was getting linked to in the forms.  I had switched to webpacker, but that caused two issues (I believe):
- Existing forms would break, since the asset pipeline version of campaign-form.js was no longer available
- The webpacker version of the file link included a digest that would change with each deploy, which would also break existing links

I solved issue one by adding the campaign-form.js file back into the asset pipeline, so that we have backwards compatibility with existing forms (although now campaign-form.js exists in two places).

I solved issue two with a snippet of code that removes the digest from the file link, so the url should stay the same between deploys.  The webpacker version also required a new config option to make sure it outputted a full URL and not just a relative link.

File: app/javascript/campaign-form.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
+//= require jquery.js
```
Comment (knutsenm) [inline app/javascript/campaign-form.js:1]: To keep the contents identical, make this file a relative symlink to `../assets/javascripts/campaign-form.js`

File: Gemfile [hunk 2]
```diff
@@ -59,10 +97,7 @@ group :development, :test do
 end
 
 group :development do
-  gem "listen", ">= 3.0.5", "< 3.2"
   gem "pry"
   gem "pry-byebug"
   gem "rack-cors"
```
Comment (knutsenm) [inline Gemfile:102]: Merge this into `group :development` above?

File: app/javascript/campaign-form.js (hunk 1 above)
Comment (jbirdjavi) [reply app/javascript/campaign-form.js:1]: Good call, although I did it the other way around.

File: Gemfile (hunk 2 above)
Comment (jbirdjavi) [reply Gemfile:102]: Unless you have a strong feeling about that, I'd rather not, since I like to keep the rails default gems separate from what we added.

**Pull request merged**